### ISSUE TITLE
Poll cluster add

### ISSUE THREAD

**Issue by jchapelle** [felixge/node-mysql#1432]
Hello, 
I was just wondering what is the "config" object passed to "poolCluster.add(config)"

Can we pass a pool object like this ?
```js
{
	connectionLimit : 100,
	host     : ip,
	port	 : port,
	user     : user,
	password : password,
	database : database
}
```

**Comment by dougwilson** [comment]
Hi @jchapelle, the README is still lacking a lot from this feature, as when the feature was contributed, it didn't come along with very many docs. The `poolCluster.add` API accepts a pool config object (https://github.com/felixge/node-mysql#pool-options). Your example is exactly correct.

**Comment by jchapelle** [comment]
Many thanks

On Tue, May 31, 2016 at 5:28 AM, Douglas Wilson <notifications@github.com>
wrote:

> Closed #1432 <https://github.com/felixge/node-mysql/issues/1432> via
> 5ad2e24
> <https://github.com/felixge/node-mysql/commit/5ad2e2426913ba27eedffc20ba183022aa18a56e>
> .
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/felixge/node-mysql/issues/1432#event-676484334>, or mute
> the thread
> <https://github.com/notifications/unsubscribe/AF4GXd_Jh9jkiH6x-6OetQS0-Qr9WqgQks5qG6rbgaJpZM4Ip7-A>
> .
>



-- 
Jean Chapelle
